### DISCUSSION TITLE
Decision to move from drake to targets

### DISCUSSION THREAD

**Discussion by liutiming** [ropensci/targets#441] [General]
Hi @wlandau, as drake is already a very successful package, I am wondering if you considered using a design similar to `testthat` and continue targets as `drake version=2.0` when you made the decision to start on targets?

**Comment by wlandau** [comment]
The changes were far too extreme, everything from the interface to storage internals to the object-oriented design of the package itself. I really needed a clean break just like `renv` broke away from `packrat` and `dplyr` broke away from `plyr`.
